src/components/Footer/Footer.module.scss

```diff
@@ -0,0 +1,54 @@
+.footer_wrapper {
```

Comment:
Why do we need to create this ?? we can use tailwind atomic classes instead to make it more understandable and scalable